### PR TITLE
front: add sticky position on stdcm map and convert rem to px

### LINKED ISSUE #9684: STDCM Map should be sticky to the top
### What happened?

When scrolling, the map doesn't stick to the top of the page, it should be fully visible when scrolling.

https://github.com/user-attachments/assets/4c86ad62-efd6-4190-a8a4-cd37b35301c2

### What did you expect to happen?

I expect the map to be sticky, as specified in [the mockup](https://www.sketch.com/s/15e0909c-f17d-4200-a770-b9fc1154fa7e/a/j4OEePa) (and as noticed by @thibautsailly) 

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to STDCM
2. Scroll (extra points if you add waypoints, so that you can completely hide the map)

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

cd4cdfb

### PR DESCRIPTION
**Pull request by theocrsb**
closes #9684

### CODE REVIEW

**Review by Akctarus — APPROVED** [review]
Tested, LGTM !

File: front/src/styles/scss/applications/stdcm/_home.scss
Comment (SharglutDev) [inline front/src/styles/scss/applications/stdcm/_home.scss:1]: We should not use floated pixels. Could you round them ?

File: front/src/styles/scss/applications/stdcm/_home.scss [hunk 1]
```diff
@@ -102,49 +102,49 @@
...
-              border: 0.031rem solid rgba(121, 118, 113, 1);
+              border-radius: 4.96px;
+              border: 0.496px solid rgba(121, 118, 113, 1);
               box-shadow:
-                0rem 0.09rem 0.06rem rgba(255, 255, 255, 0.46) inset,
-                0rem -0.09rem 0.06rem rgba(148, 145, 142, 1) inset,
-                0rem 0.06rem 0.125rem rgba(0, 0, 0, 0.25);
+                0px 1.44px 0.96px rgba(255, 255, 255, 0.46) inset,
+                0px -1.44px 0.96px rgba(148, 145, 142, 1) inset,
+                0px 0.96px 2px rgba(0, 0, 0, 0.25);
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/stdcm/_home.scss]: You can remove the `px` for `0` (for all box shadow in the file)

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested, you can resolve the last comments and merge :)

File: front/src/styles/scss/applications/stdcm/_home.scss
Comment (theocrsb) [reply front/src/styles/scss/applications/stdcm/_home.scss:1]: Done

File: front/src/styles/scss/applications/stdcm/_home.scss (hunk 1 above)
Comment (theocrsb) [reply front/src/styles/scss/applications/stdcm/_home.scss]: Done

**Pull request merged**